source/includes/scripting/fact-require-ems-modules.rst

```diff
@@ -0,0 +1,12 @@
+Node.js modules are packaged in two formats. Newer modules use the
+``ECMAScript`` standard. Older modules use the ``CommonJS`` (CJS)
```

Comment:
[question]

I think "standard" makes sense here, but I'm curious if "format" would be better as it's more consistent with the surrounding language. What do you think?